.github/workflows/test.yml

```
@@ -0,0 +1,70 @@
+name: Test of Run Command with CASL2 Sample Files
+
+on: [push]
```

CIのトリガーには、push、pull_requestをつけることをよくします。
一見、2重に走っているように思いますが、今回のPRのようなfork先からのPRでも、本家リポジトリ上でCIが実行されるようになるはずです。
そうすれば、このPR上でテストをすべてpassしていることが確認出来るようになって嬉しいです！
```suggestion
on:
  push:
  pull_request:
```